ci/build-package-set.sh

```diff
@@ -16,7 +16,7 @@ export PATH="$tmpdir/node_modules/.bin:$PATH"
 cd "$tmpdir"
 
 echo ::group::Ensure Spago is available
-which spago || npm install spago
+which spago || npm install spago@0.20.3
```

Comment:
Why do we pin the Spago version here?

Reply:
I think so. Will you find it helpful if I port the same pull request for Spago, or do you want to handle it directly?

Reply:
I'd love if you could port this same work to Spago!